src/components/HorizontalVideoCard/HorizontalVideoCard.js

```diff
@@ -0,0 +1,44 @@
+import { FaPlayCircle } from "react-icons/fa";
+import { MdOutlineClose, MdOutlineTimer } from "react-icons/md";
+import "./HorizontalVideoCard.css";
+
+export const HorizontalVideoCard = ({ video, removeHandler, handlerArgs }) => {
+  const { _id, title, creator, description, releaseDate, duration } = video;
+
+  return (
+    <article className="card card--horizontal ">
+      <div
+        className="card__close-btn"
+        title="Remove"
+        onClick={() => removeHandler(...handlerArgs)}
+      >
+        <MdOutlineClose />
+      </div>
+      <div className="card__img-container">
+        <div className="card__overlay">
+          <FaPlayCircle />
+          <span>Play</span>
+        </div>
+        <img
+          src={`https://i.ytimg.com/vi/${_id}/maxresdefault.jpg`}
+          className="img-responsive"
```

Comment:
Please be consistent with class names , you are using BEM for naming classes but at some places you are using different approach.

Reply:
Thanks for the suggestion. Will make the changes while refactoring.